website/docs/admin_hosts_blender.md

```diff
@@ -80,4 +80,26 @@ You can close new terminal. Run pip install command above again. Now should work
 
 </TabItem>
 
-</Tabs>
\ No newline at end of file
+</Tabs>
+
+## Blender Project Settings
+
+### Set Unit Scale
+Enable it to define a default unit scale. 
+- **Apply on Opening Existing Files:** enbale it to set the unit scale defined below to apply it on exiting files you open.
+- **Base File Unit Scale:** define here the default unit scale. 
+
+![Set Unit Scale](assets/project_settings_blender_unitScale.png)
+
+### Color Management (OCIO managed)
+//
+
+### Workfile Builder
+Find more informations about the Workfile Builder in the [Nuke Documentation](project_settings_nuke.md#workfile-builder).
```

Comment:
```suggestion
Find more informations about the Workfile Builder in the [Nuke Documentation](admin_hosts_nuke#workfile-builder).
```

Wrong link. Also we should probably take that workfile builder docs out of hosts, and maybe do a global section on it with some tutorials. It will be repeated a lot probably